src/extensions/data_array.js

```diff
@@ -67,24 +96,101 @@ export default () => {
       });
     };
 
+    const createInstances = observationObjects => {
+      return new Promise(resolve => {
+        db().then(models => {
+          Promise.all(observationObjects.map(observationObj => {
+            const newReq = Object.create(req);
+            newReq.body = observationObj.body;
+            return models.createInstance(observations, newReq, exclude).then(
```

Comment:
This one is going to be tricky... If any of the instance creation requests fails, we need to rollback everything. This means that we need to use a transaction for all these creations.

Create a sequelize transaction here and make `db.createInstance` take a transaction object as an argument. If no transaction is given to `db.createInstance` you'll need to create one inside it (just like it currently does). Otherwise, you'll use the given transaction.